src/platform/user/authorization/components/AcceptTermsPrompt.jsx

```diff
@@ -60,6 +60,8 @@ class AcceptTermsPrompt extends React.Component {
       return <div />;
     }
 
+    const H = `h${this.props.level || 1}`;
```

Comment:
Hey @noahgelman, could you change this variable name to be something a bit more descriptive? Making it something more explanatory would be nice for future development in this section.